src/test/kotlin/io/github/kryszak/gwatlin/http/serializers/NumberSerializerTest.kt

```diff
@@ -0,0 +1,23 @@
+package io.github.kryszak.gwatlin.http.serializers
+
+import io.github.kryszak.gwatlin.http.serializers.JsonConfigurer.json
+import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.matchers.doubles.plusOrMinus
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldStartWith
+
+class NumberSerializerTest : ShouldSpec({
```

Comment:
Make this class `internal` to stop codacy from complaining

Reply:
Fixed 👍